ivadomed/scripts/download_data.py

```diff
@@ -120,28 +119,49 @@ def unzip(compressed, dest_folder):
 
 def install_data(url, dest_folder, keep=False):
     """
-    Download a data bundle from a URL and install in the destination folder.
-    :param url: URL or sequence thereof (if mirrors).
-    :param dest_folder: destination directory for the data (to be created).
-    :param keep: whether to keep existing data in the destination folder.
-    :return: None
+    Download a data bundle from an URL and install it in the destination folder.
+
+    Usage example ::
+
+        ivadomed_download_data -d data_testing -o ivado_testing_data
+
+    Existing data bundle:
+
+        - `data_example_spinegeneric <https://github.com/ivadomed/data_example_spinegeneric/archive/r20200825.zip>`_: 10 randomly picked subject from `Spine Generic <https://github.com/spine-generic/data-multi-subject>`_. Used for Tutorial and example in Ivadomed.
```

Comment:
we should replace with an automatic generation from the DICT_URL

ideally DICT_URL would contain the description (nested dic)

Reply:
Ok so it's done after a long trial and error period see 
482a676 Visual is not really great. The issue is the following: The part that I add does not use the docstring convention so I had to improvise a bit (hence it is not a bullet list). But at least it's dynamic.

Reply:
Well small thing to correct, links are not working (I wanted to point to the repo and not the zip so I tried something but it does not work). I will fix that.

Reply:
fixed in cf89b3d